include/ignition/physics/PlaneShape.hh

```diff
@@ -132,8 +134,8 @@ namespace physics
       ///   Name to give to the PlaneShape
       /// \param[in] _normal
       ///   Normal vector for the plane
-      /// \param[in] _offset
-      ///   Offset of the plane
+      /// \param[in] _point
+      ///   The point that the plane passes through (hence defining the plane)
```

Comment:
Is this point default to the origin `Point::Zero`?

Reply:
Yea, I think the plane will pass through point zero as default.

Reply:
```suggestion
      ///   The point that the plane passes through (the origin by default)
```